README.md

```diff
@@ -84,29 +84,56 @@ This will add the specific versions of the releases that are currently deployed
 }
 ```
 
-### Deploy to local envirionment
+### Generate a manifest to only deploy certain packages
+
+```
+python .\generate_manifest.py -p Huddle.XYZ

```

Comment:
may be worth showing more than one ... just so people know if its space comma etc.

Reply:
This should have been in the other PR but I'll let it slip ;)